src/app/tests/suites/certification/Test_TC_DGGEN_1_1.yaml

```diff
@@ -43,8 +43,8 @@ tests:
       command: "readAttribute"
       attribute: "FeatureMap"
       response:
-          value: 0
           constraints:
+              contains: [0, 1]
```

Comment:
IIRC `contains` is not what you want. This will check that you get a list back and that it contains a value of 0 and 1 in that list

I don't think we have a constraint check in yaml that allow this value to be either 0 or 1. We might need PICS or something here to check for the right value under the right conditions. Or we could move away from PICS and convert this to a python test where it is conditional on what MaxPathsPerInvoke (either as session parameter or from basic info cluster) is set to.